pkg/filemanager/filemanager.go

```diff
@@ -7,6 +7,7 @@ import (
 	"io"
 	"net/url"
 	"os"
+	gp "path"
```

Comment:
minor: I would rather change a variable name instead of creating an alias for a standard package. 

Makes code harder to understand.

Reply:
I removed the alias and renamed variables/arguments.